src/lavatory/commands/purge.py

```diff
@@ -27,33 +33,31 @@ def purge(ctx, dryrun, policies_path, default, repo):
         all_repos = repo
     else:
         all_repos = before_repo_data.keys()
-    
+
     LOG.info("Applying retention policies to %s", ', '.join(all_repos))
-    for repo in all_repos:
-        policy_name = repo.replace("-", "_")
-        artifactory_repo = Artifactory(repo_name=repo)
+    for repository in all_repos:
```

Comment:
Why change the name of this variable to `repository`? I feel like `repo` is just as clear

Reply:
I changed because it is same as argument repo, confusing.